rpcd/playbooks/ironic-post-install-config.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+- name: Ironic post install config
+  hosts: ironic_all
+  tasks:
+     - name: Setup neutron network
+      os_network:
+        shared: yes
+        provider_physical_network: "tftp"
+        provider_network_type: "flat tftp"
+
+    - name: Setup neutron subnet
+      os_subnet:
+        name: "ironic-ipmi"
+        allocation_pool_start: "172.29.233.0"
+        allocation_pool_end: "172.29.234.0"
+        dns_nameservers: "4.4.4.4 tftp 172.29.232.0/22"
+
+    - name: Gather cleaning network UUID
+      shell: "neutron net-list | grep tftp | awk '{print $2}'"
```

Comment:
You might consider using the new openstack commands if they are available:
openstack network list --name tftp -f value -c ID
neutron commands are in a deprecation phase.